packages/vuetify/src/labs/VDateInput/__tests__/VDateInput.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { VDateInput } from '../VDateInput'
+
+// Utilities
+import { mount } from '@vue/test-utils'
+import { createVuetify } from '@/framework'
+
+global.ResizeObserver = require('resize-observer-polyfill')
```

Comment:
How do we do this in other places? Surely this isn't the first time we're unit testing a component that utilizes the resize observer.

Reply:
[This](https://github.com/vuetifyjs/vuetify/blob/04551e2c54454ed9ae0221b6e9fbb0789912342e/packages/docs/src/pages/en/getting-started/unit-testing.md?plain=1#L80) was the only place where `resize-observer-polyfill` was referenced in the entire codebase. So I just followed this

Reply:
I converted it to `VDateInput.spec.browser.tsx` and removed the necessity to mock ResizeObserver